src/devices/Bmxx80/tests/Bmxx80.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <LangVersion>latest</LangVersion>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <RootNamespace>Iot.Device.Imu.Tests</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+    <PackageReference Include="xunit" Version="2.4.1" />
```

Comment:
NIT: we don't have to fix it on this PR, but these references to xunit on all of our UnitTests projects should be against the same version instead of what we have now where different projects depend on different versions. Perhaps we should even add some infrastructure so that if this is a unit test project then these references get automatically added for you so that we control and make sure all versions match.